test/parallel/test-whatwg-url-searchparams-sort.js

```diff
@@ -55,12 +55,23 @@ const { test, assert_array_equals } = common.WPT;
 /* eslint-enable */
 
 // Tests below are not from WPT.
-;[
+
+// Test bottom-up iterative stable merge sort
+const tests = [{input: '', output: []}];
+for (let i = 0; 50 > i; i++) {
+  tests[0].input += 'a=b&';
+  tests[0].output.push(['a', 'b']);
```

Comment:
Hmm..so looks like this doesn't actually catch a mistake in the implementation because we are sorting identical pairs...maybe we can generate the keys on the fly, pseudo-shuffle them, then compared to another original copy to make sure they are sorted properly?

```js
const tests = [{input: '', output: []}];
const pairs = [];
for (let i = 10; i < 60; i++) {
  pairs.push([`a${i}`, 'b']);
  tests[0].output.push([`a${i}`, 'b']);
}
tests[0].input = pairs.sort(() => Math.random() > 0.5).map((pair) => pair.join('=')).join('&');
```